content/en/network_performance_monitoring/network_page.md

```diff
@@ -150,6 +150,36 @@ Select any row from the data table to see associated logs, traces, and processes
 
 {{< img src="network_performance_monitoring/network_page/flow_details.png" alt="Aggregate Connection Details"  style="width:80%;">}}
 
+### Rollups
+
+Rolled up traffic (either port or IP) are displayed with `*`. They represent multiple ports or ips being condensed for a given flow.  
+
+##### IP Addresses:
+
+Rolled up when there are connections to multiple public IPs. For example:
```

Comment:
I'm not sure if it's fully necessary since it may be obvious & the example underneath illustrates it, but it might be worth specifying that IP addresses are rolled up when multiple public IPs use the same port